packages/adblocker/src/data-view.ts

```diff
@@ -20,6 +20,9 @@ export const EMPTY_UINT32_ARRAY = new Uint32Array(0);
 // Check if current architecture is little endian
 const LITTLE_ENDIAN: boolean = new Int8Array(new Int16Array([1]).buffer)[0] === 1;
 
+// TextEncoder doesn't need to be recreated every time unlike TextDecoder
+const TEXT_ENCODER = new TextEncoder();
```

Comment:
not sure if we should do that or create a new instance for every call to `encode`

Reply:
Benchmark results here: [(link)](https://perf.link/#eyJpZCI6InYwMWNuZzRlbzllIiwidGl0bGUiOiJTaG91bGQgSSB1c2UgVEVYVF9FTkNPREVSIGdsb2JhbD8iLCJiZWZvcmUiOiJjb25zdCByYW5kb21TdHJpbmdzID0gW11cbmNvbnN0IFNJWkUgPSAxICogMTAwMFxuZm9yIChsZXQgaSA9IDA7IGkgPCBTSVpFOyBpKyspIHtcbiAgcmFuZG9tU3RyaW5ncy5wdXNoKChNYXRoLnJhbmRvbSgpICogRGF0ZS5ub3coKSkudG9TdHJpbmcoMzIpLnNsaWNlKDAsIDYpKVxufVxuY29uc3QgYnVmZmVyID0gbmV3IFVpbnQ4QXJyYXkoU0laRSkiLCJ0ZXN0cyI6W3sibmFtZSI6IlRFWFRfRU5DT0RFUiBnbG9iYWwiLCJjb2RlIjoiY29uc3QgVEVYVF9FTkNPREVSID0gbmV3IFRleHRFbmNvZGVyKCk7XG5mb3IgKGxldCBpID0gMCwgcG9zID0gMDsgaSA8IHJhbmRvbVN0cmluZ3MubGVuZ3RoOyBpKyspIHtcbiAgY29uc3QgeyB3cml0dGVuIH0gPSBURVhUX0VOQ09ERVIuZW5jb2RlSW50byhyYW5kb21TdHJpbmdzW2ldLCBidWZmZXIuc3ViYXJyYXkocG9zKSlcbiAgcG9zICs9IHdyaXR0ZW5cbn0iLCJydW5zIjpbMTg3NSwxNjI1LDIwMDAsMTg3NSwxODc1LDE2MjUsMjAwMCwxNzUwLDE3NTAsMTg3NSwyMDAwLDIwMDAsMTYyNSwxODc1LDE4NzUsMjAwMCwxNzUwLDIwMDAsMTc1MCwyMDAwLDE3NTAsMTg3NSwxODc1LDE4NzUsMTg3NSwxODc1LDIwMDAsMTg3NSwyMDAwLDE4NzUsMjAwMCwxNzUwLDE4NzUsMTg3NSwxNzUwLDIwMDAsMTg3NSwxNzUwLDIwMDAsMTg3NSwxNzUwLDE4NzUsMjAwMCwxNjI1LDE4NzUsMjAwMCwyMDAwLDE3NTAsMTg3NSwyMDAwLDE4NzUsMTUwMCwxODc1LDIwMDAsMTg3NSwxNzUwLDE4NzUsMTg3NSwxODc1LDE3NTAsMTg3NSwxODc1LDE3NTAsMTg3NSwyMDAwLDIwMDAsMTYyNSwyMDAwLDE4NzUsMjAwMCwxODc1LDE4NzUsMjAwMCwxODc1LDE3NTAsMTg3NSwyMDAwLDE4NzUsMTg3NSwyMDAwLDIwMDAsMjAwMCwxNjI1LDIwMDAsMTc1MCwyMDAwLDE3NTAsMjAwMCwyMDAwLDE4NzUsMTYyNSwyMDAwLDIwMDAsMTc1MCwxNjI1LDE2MjUsMjAwMCwyMDAwLDE4NzUsMTg3NV0sIm9wcyI6MTg3MH0seyJuYW1lIjoiQ3JlYXRlIHRleHQgZW5jb2RlciBldmVyeSB0aW1lIiwiY29kZSI6ImZvciAobGV0IGkgPSAwLCBwb3MgPSAwOyBpIDwgcmFuZG9tU3RyaW5ncy5sZW5ndGg7IGkrKykge1xuICBjb25zdCB7IHdyaXR0ZW4gfSA9IG5ldyBUZXh0RW5jb2RlcigpLmVuY29kZUludG8ocmFuZG9tU3RyaW5nc1tpXSwgYnVmZmVyLnN1YmFycmF5KHBvcykpXG4gIHBvcyArPSB3cml0dGVuXG59IiwicnVucyI6WzEyNTAsMTEyNSwxMjUwLDEyNTAsMTM3NSwxMTI1LDEzNzUsMTI1MCwxMjUwLDEyNTAsMTUwMCwxMjUwLDEyNTAsMTI1MCwxMjUwLDEyNTAsMTEyNSwxMjUwLDEyNTAsMTM3NSwxMjUwLDExMjUsMTM3NSwxMzc1LDEyNTAsMTI1MCwxMzc1LDEyNTAsMTI1MCwxMjUwLDEzNzUsMTEyNSwxMjUwLDEyNTAsMTI1MCwxMjUwLDEyNTAsMTI1MCwxMzc1LDEyNTAsMTI1MCwxMjUwLDEyNTAsMTEyNSwxMjUwLDEyNTAsMTI1MCwxMjUwLDEyNTAsMTM3NSwxMjUwLDExMjUsMTI1MCwxMzc1LDEyNTAsMTEyNSwxMjUwLDEyNTAsMTI1MCwxMjUwLDEyNTAsMTI1MCwxMTI1LDEyNTAsMTI1MCwxMzc1LDEyNTAsMTI1MCwxMzc1LDEzNzUsMTEyNSwxMjUwLDEzNzUsMTI1MCwxMjUwLDEyNTAsMTI1MCwxMTI1LDEyNTAsMTI1MCwxMzc1LDEyNTAsMTI1MCwxMjUwLDEyNTAsMTM3NSwxMjUwLDEyNTAsMTI1MCwxMjUwLDExMjUsMTI1MCwxMzc1LDEyNTAsMTI1MCwxMTI1LDEzNzUsMTM3NSwxMjUwLDEyNTBdLCJvcHMiOjEyNjB9XSwidXBkYXRlZCI6IjIwMjQtMTItMTFUMTA6MjM6NDQuOTQ1WiJ9)